db/db_users_table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE users (
```

Comment:
let's have all quiries be on the same folder. so, we could structure like this:

db 
- quiries
- sqlc

in the future, we will put something like an extension from sqlc generation.